src/stream/src/task/stream_manager.rs

```diff
@@ -181,17 +181,20 @@ impl LocalStreamManager {
         rx.await.unwrap()
     }
 
-    pub async fn sync_epoch(&self, epoch: u64) -> Vec<LocalSstableInfo> {
-        dispatch_state_store!(self.state_store(), store, {
-            match store.sync(Some(epoch)).await {
-                Ok(_) => store.get_uncommitted_ssts(epoch),
-                // TODO: Handle sync failure by propagating it back to global barrier manager
-                Err(e) => panic!(
+    pub async fn sync_epoch(&self, epoch: u64) -> (Vec<LocalSstableInfo>, bool) {
+        (
+            dispatch_state_store!(self.state_store(), store, {
+                match store.sync(Some(epoch)).await {
+                    Ok(_) => store.get_uncommitted_ssts(epoch),
+                    // TODO: Handle sync failure by propagating it back to global barrier manager
+                    Err(e) => panic!(
                     "Failed to sync state store after receiving barrier prev_epoch {:?} due to {}",
                     epoch, e
                 ),
-            }
-        })
+                }
+            }),
+            true,
```

Comment:
Why we need it if it's always true?

Reply:
Because It is only a temporary version. I only add is_sync and do some things in barrier_manager.I will add other states in the future.

Reply:
Okay, I thought the pr was ready to review. 🤣 Please convert this to draft PR if it's not ready yet.